File: TodoApp.WebApp/Client/src/modules/action-item/model/actionItem.ts
export class IActionItem {
    public id: number;
    public categoryId: number;
    public content: string;
    public start: string;
    public end: string;
    public status: number;
    public isDone: boolean;

    constructor() {
        this.id = 0;
        this.categoryId = 0;
        this.content = '';
        this.start = '';
        this.end = '';
        this.status = 0;
        this.isDone = false;
    }
}

File: TodoApp.Database/Entities/ActionItem.cs
namespace TodoApp.Database.Entities
{
    public class ActionItem
    {
        public long Id { get; set; }
        public int CategoryId { get; set; }
        public string Content { get; set; }
        public short Status { get; set; }
        public int UserId { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        public DateTime Created { get; set; }
        public DateTime Updated { get; set; }

        public virtual User User { get; set; }
        public virtual Category Category { get; set; }
    }
}